src/lib.rs

```diff
@@ -35,10 +35,42 @@ impl<'a> Image<'a> {
     }
 }
 
+/// Scales the alpha to the downscaled texture to preserve the overall alpha coverage.
+///
+/// If alpha cutoff is specified, any alpha value above it is considered visible
+/// of which the percentage of visible texels will be. Otherwise, visibility is considered
+/// a linear sum of the alpha values instead and the source and target alpha coverage are calculated
+/// the same way.
```

Comment:
Same remark about line-length and reflowing. If the 3rd sentence is as long as it is, the first could be longer as well? I let my editor help me maintain a max length of 100 (until `rustfmt`, years and years after it being unstable afaik, may hopefully one day be able to do this).

Reply:
Yeah that would have been nice, also just for consistency between different contributors